docs/source/onnx.rst

```diff
@@ -9,11 +9,10 @@ Here is a simple script which exports a pretrained AlexNet as defined in
 torchvision into ONNX.  It runs a single round of inference and then
 saves the resulting traced model to ``alexnet.onnx``::
 
-    from torch.autograd import Variable
     import torch.onnx
     import torchvision
 
-    dummy_input = Variable(torch.randn(10, 3, 224, 224)).cuda()
+    dummy_input = torch.randn(10, 3, 224, 224).cuda()
```

Comment:
the idomatic way to do this now is `torch.randn((10, 3, 224, 224), device='cuda')`, would be great if you could change this :)